### DISCUSSION TITLE
[done] Как в быстрой форме сделать Tabletext форматированным.

### DISCUSSION THREAD

**Discussion by aityomin** [totumonline/totum-ru-issues-and-discussions#187] [Есть идея, а что если...]
Имеется ввиду переносы строк, части текста разного формата...

**Привет!**
_Привет!_
### Привет!

как-то так...

**Comment by ttm-alexeypolunin** [comment]
Не делали такую историю, возможно в PRO будет форматированный текст.

**Reply by aityomin** [reply]
А может это можно каким-нибудь способом обойти? Дополнительное поле ввести, например?

**Reply by ttm-alexeypolunin** [reply]
Епрст — tabletext (я не туда посмотрел), возможно сделаем сегодня.

**Reply by aityomin** [reply]
Спасибо!

**Reply by ttm-alexeypolunin** [reply]
Обновите, проверьте

**Reply by aityomin** [reply]
А на демо-сервере работает?

**Reply by ttm-alexeypolunin** [reply]
Нет, и мы на demo заблокируем формы и анонимные таблицы, что бы там наркоту не продавали.
